package.json

```diff
@@ -54,7 +54,7 @@
     "@atomist/slack-messages": ">=1.1.0"
   },
   "devDependencies": {
-    "@atomist/automation-client": "^1.1.0",
+    "@atomist/automation-client": "1.2.0-generate-repo-id.20181220113107",
```

Comment:
Should we wait until atomist/automation-client#434 has been merged and use that version?